internal/runners/commit/commit.go

```diff
@@ -140,9 +145,26 @@ func (c *Commit) Run() (rerr error) {
 		return errs.Wrap(err, "Could not update local build script")
 	}
 
-	pg.Stop(locale.T("progress_success") + "\n")
+	pg.Stop(locale.T("progress_success"))
 	pg = nil
 
+	// Solve runtime
+	_, rtCommit, err := runtime.Solve(c.auth, c.out, c.analytics, c.proj, &stagedCommitID, target.TriggerCommit, c.svcModel, c.cfg, runtime.OptNone)
+	if err != nil {
+		return errs.Wrap(err, "Could not solve runtime")
+	}
+
+	// Get old buildplan.
+	rtTarget := target.NewProjectTarget(c.proj, &stagedCommitID, target.TriggerCommit)
+	commit, err := bp.FetchCommit(localCommitID, rtTarget.Owner(), rtTarget.Name(), nil)
+	if err != nil {
+		return errs.Wrap(err, "Failed to fetch build result")
+	}
+	oldBuildPlan := commit.BuildPlan()
```

Comment:
We should wrap this in a progress spinner for solving.

Reply:
`runtime.Solve()` has its own `Resolving Dependencies` spinner (as you can see in the output at the top of this PR). Is that sufficient?

Reply:
@mitchell-as it doesn't cover the FetchCommit call afterwards, which also is effectively a solve. I believe you can disable the UI from `runtime.Solve()` with a flag (`runtime.NoUI` or something like that).

This will be especially appropriate for the runtime refactor, which eliminates `runtime.Solve()` in favour of a method that does not produce any UI.